File: misc/001-Two Sum/main.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize)
{
    int i, j;
    for(i = 0; i < numsSize-1; i++)
    {
        for(j = i+1; j < numsSize; j++)
        {
            if(nums[i] + nums[j] == target)
            {
                int* ret = malloc(2*sizeof(int));
                *returnSize = 2;
                ret[0] = i;
                ret[1] = j;
                return ret;
            }
        }
    }

    *returnSize = 0;
    return NULL;
}

File: misc/001-Two Sum/main.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        num_len = len(nums)
        for i in range(num_len-1):
            for j in range(i+1,num_len):
                if((nums[i] + nums[j]) == target):
                    return [i,j]

#if __name__ == "__main__":
#    print(Solution().twoSum([2,7,11,15], 9))